library/src/scripts/headers/TitleBar.tsx

```diff
@@ -100,9 +99,16 @@ export class TitleBar extends React.Component<IProps, IState> {
         const showHamburger = isCompact && !this.state.openSearch && !!hamburger;
         const classesMeBox = meBoxClasses();
 
+        const localNavLinks = navigationVariables()[`${getCurrentLocale()}`]
+            ? "data" in navigationVariables()[`${getCurrentLocale()}`]
+            : null;
+        const navigationData = localNavLinks
+            ? navigationVariables()[`${getCurrentLocale()}`].data
+            : defaultNavigationData().data;
+
         const navLinks = navigationLinks
-            ? dummyNavigationData().data.concat(dummyStorybookNavigationData().data)
-            : dummyNavigationData().data;
+            ? defaultNavigationData().data.concat(dummyStorybookNavigationData().data)
+            : navigationData;
```

Comment:
Shouldn't it be?
```suggestion
            : navigationData().data;
```

Reply:
No, because we are getting ().data from navigationData, please see line 106,107